README.md

```diff
@@ -1,13 +1,11 @@
-
-
 <!-- toc -->
 
```

Comment:
Could we have somewhere a shell script to build and copy the 2 crates in the right place? (in order to get ready to dev)

Reply:
Done in b2db1c8